eng/scripts/Create-APIReview.ps1

```diff
@@ -14,30 +14,31 @@ Param(
     [Parameter(Mandatory=$True)]
     [string] $DefaultBranch,
     [Parameter(Mandatory=$True)]
-    [string] $ConfigFileDir
+    [string] $ConfigFileDir,
+    [Parameter(Mandatory=$True)]
+    [string] $ParserPath,
+    [Parameter(Mandatory=$True)]
+    [string] $SourcePath
 )
 
 Write-Host "$PSScriptRoot"
 . (Join-Path $PSScriptRoot .. common scripts common.ps1)
 $createReviewScript = (Join-Path $PSScriptRoot .. common scripts Create-APIReview.ps1)
-Set-Location $PSScriptRoot
 
-Write-Host "Creating API review artifact for $ArtifactName"
-New-Item -ItemType Directory -Path $OutPath/$ArtifactName -force
+$apiviewSettings = Join-Path $SourcePath "ApiViewSettings.json"
+if (!(Test-Path $apiviewSettings))
+{
+    Write-Host "ApiViewSettings.json file is not found in $($SourcePath). APIView settings file is required to generate API review file."
+    exit 1
+}
 
-$gitroot = Join-Path $PSScriptRoot .. ..
-Write-Host "Get-ApiViewCommandLine.ps1 $gitroot $ArtifactName"
-$cmdLine = & $PSScriptRoot/Get-ApiViewCommandLine.ps1 $gitroot $ArtifactName
-Write-Host "Executing clang++ command:"
-Write-Host $cmdLine
-$cmd, $cmdArgs = $cmdLine -split ' '
-# Get-ApiViewCommandLine.ps1 returns a string representing a clang++ command that needs to be run, e.g.
-# clang++ <space separated list of header files> -Xclang -ast-dump -I <space separated list of header files>
-# ApiView expects a zip of this ast as the format for a C++ language artifact.
-& $cmd $cmdArgs > clangAstOutput
+Write-Host "Creating API review artifact for $($ArtifactName)"
+New-Item -ItemType Directory -Path $OutPath/$ArtifactName -force
+$parentPath = Split-Path $ParserPath  -Parent
+Write-Host "Contents in $($parentPath)"
+Get-ChildItem -Path $parentPath -Recurse
 
-Compress-Archive -Path clangAstOutput -DestinationPath $OutPath/$ArtifactName/$ArtifactName
-Rename-Item $OutPath/$ArtifactName/$ArtifactName.zip -NewName "$ArtifactName.cppast"
+& $ParserPath -o $OutPath/$ArtifactName/$ArtifactName.json $SourcePath
```

Comment:
Is there a way to specify package version in the invocation? It looks like the existing cppast based API reviews seem to have a version number in the API reviews.

Reply:
Let me check this.

Reply:
Are you referring package version or parser version? if it's package version: where is information present within package?

Reply:
Package version - parser version isn't interesting to API reviewers.

The <root>\Scripts\Get-PkgVersion.ps1 will get the package name for a given service directory and package name and return it as a string.

Reply:
This will be taken care in a follow up PR.